src/test/kotlin/study/StringCalculatorTest.kt

```diff
@@ -0,0 +1,40 @@
+package study
+
+import StringCalculator
+import io.kotest.core.spec.style.StringSpec
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+
+class StringCalculatorTest : StringSpec({
+    "plusTest" {
+        assertThat(StringCalculator.plus(1, 2) == 3).isTrue()
```

Comment:
kotest를 활용하여 테스트를 잘 작성해주셨습니다 👍 
kotest의 assertion matchers도 활용해보면 어떨까요?

- https://kotest.io/docs/assertions/assertions.html

```suggestion
        StringCalculator.plus(1, 2) shouldBe 3
```